frontend/server/src/main/java/org/pytorch/serve/http/api/rest/ManagementRequestHandler.java

```diff
@@ -48,9 +49,12 @@
  */
 public class ManagementRequestHandler extends HttpRequestHandlerChain {
 
+    ConfigManager configManager;
+
     /** Creates a new {@code ManagementRequestHandler} instance. */
     public ManagementRequestHandler(Map<String, ModelServerEndpoint> ep) {
         endpointMap = ep;
+        configManager = ConfigManager.getInstance();
```

Comment:
Make member variable private
```suggestion
        private configManager = ConfigManager.getInstance();
```